BedrockServer.cpp

```diff
@@ -1122,7 +1140,8 @@ void BedrockServer::_reply(BedrockCommand& command) {
     }
 }
 
-void BedrockServer::suppressCommandPort(bool suppress, bool manualOverride) {
+void BedrockServer::suppressCommandPort(const string& reason, bool suppress, bool manualOverride) {
+    SINFO((suppress ? "Suppressing" : "Clearing") << " command port due to: " << reason);
```

Comment:
Can you put this line underneath the comment?  See [here](https://github.com/Expensify/Style-Guide/blob/master/php.md#blank-lines)